src/Psl/Str/convert_encoding.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Str;
+
+use function mb_convert_encoding;
+
+/**
+ * Convert character encoding of the giving string.
+ *
+ * @psalm-return string Converts the character encoding of $string to $to_encoding from optionally $from_encoding.
+ *
+ * @psalm-pure
+ */
+function convert_encoding(string $string, string $from_encoding, string $to_encoding): ?string
```

Comment:
we need to verify that both $from_encoding and $to_encoding are valid before calling mb_convert_encoding

Reply:
How we can verify both $from_encoding and $to_encoding, any idea?

Reply:
list this:

```php
use Psl;
use Pls\Arr;

Psl\invariant(Arr\contains(mb_list_encodings(), $from_encoding), '$from_encoding is invalid.');
Psl\invariant(Arr\contains(mb_list_encodings(), $to_encoding), '$to_encoding is invalid.');
```